holoviews/core/spaces.py

```diff
@@ -478,6 +478,8 @@ def clone(self, callable=None, **overrides):
 
 
     def __call__(self, *args, **kwargs):
+        # Nothing to do for callbacks that accept no arguments
+        if not args and not kwargs: return self.callable()
```

Comment:
If the inner callable takes no arguments at all, there is nothing to do but call it and return. Memoization makes no sense as there is only an empty tuple to memoize on - the output is constant for all time so might as well use an element. There can be no streams to worry about at this level as there are no args or kwargs either.